packages/sdk/src/libs/stores/index.ts

```diff
@@ -11,4 +12,9 @@ export interface Store<I, D> {
   values: () => Promise<Array<D>>;
   keys: () => Promise<Array<I>>;
   size: () => Promise<number>;
+  clear: () => Promise<void>;
+  // these fields are for stores with sorted data
+  between: (a: I, b: I) => Promise<Array<D>>;
+  slice: (id: I, length: number) => Promise<Array<D>>;
+  driver?: any;
```

Comment:
added some additional functions on stores. any ones that cant be directly implemented by the store should just return empty array or throw error ( if typescript will compile)